docs/getting-started/supported-languages-and-tools.md

```diff
@@ -116,6 +116,13 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td>-</td>
       <td>-</td>
     </tr>
+    <tr>
+      <td>Dart</td>
+      <td><a href="https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli">dartanalyzer</a></td>
```

Comment:
It seems they have the `main` repo as the default and not master. They appear to be similar, so maybe link to the main with https://github.com/dart-lang/sdk/tree/main/pkg/analyzer_cli

Reply:
Well spotted @vascorsd!

Reply:
Very nice, fixed
https://github.com/codacy/docs/pull/996/commits/645ef64c72be55c7dc3dde5b89d90ad76da55bb9